apps/src/templates/instructions/CommitsAndReviewTab.jsx

```diff
@@ -105,6 +107,64 @@ const CommitsAndReviewTab = props => {
     }
   };
 
+  const deleteCodeReviewComment = async (commentId, onSuccess, onFailure) => {
```

Comment:
So if you want to take a look at how I handled the resolved comment state, I was hoping we could do something similar here. Store an isDeleted on the state of the comment and on delete success render null instead of the comment.

Reply:
It avoids having to do all this complicated logic with finding it in the top-level state.

Reply:
Ah yeah, that seems a lot more straightforward. I'll update this.